run.py

```diff
@@ -146,6 +156,10 @@ def app_mute(n_clicks):
 
 @app.callback(Output('tabs-content', 'children'), [Input('tabs', 'value')])
 def render_content(tab):
+    if(tab == 'calendar-tab'):
+        CalendarTimeObj.monthname = datetime.datetime.now().strftime("%B")
+        CalendarTimeObj.year = datetime.datetime.now().year
+        CalendarTimeObj.month = datetime.datetime.now().month
```

Comment:
Maybe dedicated class function that does it would be better.
Just calling e.g. CalendarTimeObj.set_current_time() would be more clear